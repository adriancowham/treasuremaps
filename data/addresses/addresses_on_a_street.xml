<posts>
<item rdf:about="http://sacramento.craigslist.org/gms/2339832892.html"> 
<title><![CDATA[furnitures, bedrooms, living rooms, antiques, many other appliances (p st sac)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339832892.html</link> 
<description><![CDATA[  I am having a garage sale on this Saturday 4/23/2011 (from 10:30 am to 6:00 pm) in the parking lot behind my appartement complex on 2420 P st (apt 6) Sacramento Ca 95816...
<br> 

<br> 
 A lot of wonderfull stuff sold for moving price; you know what I mean... Most of the furniture and appliances is almost brand new as it was used only by my parents for 2 months before they traveled and then just got stored, waiting the lucky buyer..... Don't miss it
<br> 

<br> 
 I might also be able to sell earlier on Friday if you are really interested so you can reach me on 916 833 6928 Ramzi / Mike.
<br> 

<br> 
 
<br> 

<br> 

<br> 

<br> 
Sacramneto, Davis, west sacramento, woodland, elk grove, natomas, carmicheal, north highlands, roseville, vacaville, dixon, california<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=p st sac -->Location: p st sac
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T19:24:46-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339832892.html</dc:source> 
<dc:title><![CDATA[furnitures, bedrooms, living rooms, antiques, many other appliances (p st sac)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T19:24:46-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339424146.html"> 
<title><![CDATA[Saturday 23rd & Sunday 24th Sale lots of deals !!! (citrus heights) $1]]></title> 
<link>http://sacramento.craigslist.org/gms/2339424146.html</link> 
<description><![CDATA[7906 Ashmont Street
<br>
This Weekend the 23rd & 24th !!!
<br>
<br>
Lots of great stuff. dont miss out.......
<br>
Google it Bing it put it in your nav unit and find a way here. 
<br>
<br>

<a href="http://photobucket.com" target="_blank" rel="nofollow"><img src="http://i895.photobucket.com/albums/ac160/irone916/craigslist/IMG_0707.jpg" border="0"></a>
<br>
<br>
<a href="http://photobucket.com" target="_blank" rel="nofollow"><img src="http://i895.photobucket.com/albums/ac160/irone916/craigslist/IMG_0708.jpg" border="0"></a>
<br>

<br>

<br>

<br>

<br>

<br>
FAMILY GARAGE SALE ESTATE SALE. WE HAVE FANTASTIC ITEMS INCLUDING BABY GEAR, FURNITURE, ELECTRONICS, HOUSEWARES, DECOR, TOYS, TOOLS, KOOLERS, AND TONS OF OTHER ITEMS. COME EARLY FOR THE BEST SELECTIONS COME EARLY. 
<br>

<br>
 sale with lots of stuff, dishes, antiques, sewing supplies, patterns, baby items, clothes, purses, books...furniture, new stuff old stuff. 
<br>

<br>
Begins at 9:00AM and Ends around 3. This Estate Sale contains many items to include: 
<br>
HOUSEHOLD ITEMS! ( Full Kitchenware, Dishware, linens, All household goods). LOTS OF SEASONAL (HOLIDAY) ITEMS (indoor/outdoor) MUST GO! EVERYTHING MUST GO!
<br>

<br>

<br>
We will have bose sound systems, new shower heads, lots of electronics, ps3, ps2, xbox 360 and wii guitar hero and rockband systems, stainless steel bbq, tvs, mattress pads, window blinds by hunter douglas, levolor and everwood, spaceheater, room coolers,  logitch harmony remote, dyson, cordless phones, slot machine, tools, craftman, kobalt, husky, snap-on, 
<br>

<br>
<div align="center"><a href="http://208.43.206.219/" target="_blank" rel="nofollow"><img src="http://208.43.206.219/Counter-Basic.cod?id=1f337858-c64d-4480-9b4a-43b2534eb705" border="0" style="border: 1px solid #ececec;"></a><br><a href="http://208.43.206.219/" target="_blank" rel="nofollow">Free Hit Counter Analytics!</a></div>
<br>
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=citrus heights -->Location: citrus heights
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T15:08:45-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339424146.html</dc:source> 
<dc:title><![CDATA[Saturday 23rd & Sunday 24th Sale lots of deals !!! (citrus heights) $1]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T15:08:45-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2339383008.html"> 
<title><![CDATA[EVERY SATURDAY  (Arden-Arcade)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339383008.html</link> 
<description><![CDATA[We will be having a yard sale every Saturday.
<br>
We always have new stuff for reasonable prices.
<br>
Big screen TVs, China Cabinets, Toys, Shoes, and Clothes too much to list.
<br>
Make sure to stop by and say Hi...
<br>
Address: 2125 Violet St. Sacramento, Ca. 95825
<br>
Call if you get lost: 916-477-7133 or 916-254-2615
<br>
Time: 8am til it's all gone.<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Arden-Arcade -->Location: Arden-Arcade
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T14:47:19-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339383008.html</dc:source> 
<dc:title><![CDATA[EVERY SATURDAY  (Arden-Arcade)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T14:47:19-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2339341965.html"> 
<title><![CDATA[2 Family Garage Sale Fri&Sat 9a-3p (Vacaville)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339341965.html</link> 
<description><![CDATA[180 East Hemlock St,  X- Dobbins, Fri & Sat,  9am-3pm - NO EARLY BIRDS PLZZZZZZ
<br> 

<br> 
Lg Executive Desk (excellant condi), Queen hideabed sofa, Swivel/Rocker Recliner, Wood Dresser and End Table, Wood Head/Fooot DLB Bed Frame,
<br> 
4ft floursent lights & bulbs, Off Road MT. Skateboard with brake, Boys clothes, girls shoe(like new, sz 6), Humidifier, Lots & Lots of Toys, books, older baseball cards and figures, and other misc items.....to much to list!!!
<br> 

<br> 

<br> 

<br> 
  <!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Vacaville -->Location: Vacaville
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T14:27:38-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339341965.html</dc:source> 
<dc:title><![CDATA[2 Family Garage Sale Fri&Sat 9a-3p (Vacaville)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T14:27:38-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2350122432.html"> 
<title><![CDATA[FRI 4/29 & SAT 4/30  (WOODLAND)]]></title> 
<link>http://sacramento.craigslist.org/gms/2350122432.html</link> 
<description><![CDATA[Spring cleaning! I have clothes (cheap) for   girls ages 2-13 / boys ages 3months -2  / womens plus size / mens plus size.   carseats / stroller for and infant carseat / baby toys / toys  / baby items / bottles / nic nacs / older computer / roler blades /  plastic play structure / smoker  and much, much, more. 
<br> 

<br> 
Also have:
<br> 
Bud light neon sign (new)  58" W x 16" H
<br> 
2 neon light up menu bords (with markers)
<br> 
Rockstar  table umbrella (new) big $125.00 crank handle/ tilt option
<br> 

<br> 
57 5th Street.  (cross streets are Beamer and clover) by east st
<br> 
Friday 9:00am or earlier - mid afternoon early evening(No later than 4:00pm)
<br> 
Saturday 7:30am -11:30 Noon latest.<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=WOODLAND -->Location: WOODLAND
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T17:11:40-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2350122432.html</dc:source> 
<dc:title><![CDATA[FRI 4/29 & SAT 4/30  (WOODLAND)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T17:11:40-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2349852738.html"> 
<title><![CDATA[Garage Sale (Fair Oaks)]]></title> 
<link>http://sacramento.craigslist.org/gms/2349852738.html</link> 
<description><![CDATA[Garage Sale Sat 04/30/2011                    HOURS NORMAL DAYTIME BUSINESS HOURS
<br> 

<br> 
8057 Sierra St. Fair Oaks Ca 95628            FURNITURE, BOOKS, CLOCKS, ANTIQUES, COLLECTIBLES, BOARD GAMES, Etc. parking on main<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Fair Oaks -->Location: Fair Oaks
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T14:38:58-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2349852738.html</dc:source> 
<dc:title><![CDATA[Garage Sale (Fair Oaks)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T14:38:58-07:00</dcterms:issued> 
</item> 

</posts>