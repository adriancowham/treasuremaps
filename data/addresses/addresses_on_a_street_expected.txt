2420 P st
7906 Ashmont Street
2125 Violet St
180 East Hemlock St
57 5th Street
8057 Sierra St